app/middlewares/validatonSchemas/users.js

```diff
@@ -0,0 +1,54 @@
+const { user: UserModel } = require('../../models/index');
```

Comment:
It is not necessary to indicate the index file. You can do the request like : `const { user: UserModel } = require('../../models');` due to for default the require find the index file.